packages/valory/skills/dynamic_nft_abci/io_/load.py

```diff
@@ -40,18 +40,13 @@
 class PNGLoader(AbstractLoader):
     """A PNG files Loader."""
 
-    def load_single_file(self, path: str) -> NativelySupportedSingleObjectType:
-        """Read an image from a PNG file.
-
-        :param path: the path of the png.
-        :return: the image object.
-        """
-        try:
-            return Image.open(path)
-        except FileNotFoundError as e:  # pragma: no cover
-            raise IOError(f"File {path} was not found!") from e
-        except (UnidentifiedImageError, ValueError, TypeError) as e:  # pragma: no cover
-            raise IOError("The provided png could not be opened and identified!") from e
+    def load_single_object(
+        self, serialized_object: str
+    ) -> NativelySupportedSingleObjectType:
+        """Load a single object."""
+        mode, width, height, data = serialized_object.split(":")
+        size = (int(width), int(height))
+        return Image.frombytes(mode, size, bytes.fromhex(data))
```

Comment:
Sorry for the inconvenience here, the thing is once the ipfs con was introduced, we were no longer dealing with files here, and it didn't make sense to keep calling it like that.